src/rf/apps/core/models.py

```diff
@@ -230,10 +243,6 @@ class LayerImage(Model):
         max_length=255,
         help_text='Filename of original file',
     )
-    s3_uuid = UUIDField(
```

Comment:
https://docs.djangoproject.com/en/1.8/ref/models/fields/#uuidfield
http://www.postgresql.org/docs/8.3/static/datatype-uuid.html
The uuid field type also prevents a user from creating a malicious post that uses a non-uuid since the uuid field enforces the correct properties of a uuid.